microcosm_pubsub/chain/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class AttributeNotFound(Exception):
+    def __init__(self, parent, attribute):
+        super().__init__(f"Failed to find attribute `{attribute}` on `{parent}`")
+
+
+class ContextKeyNotFound(Exception):
+    def __init__(self, key_error: KeyError, func):
+        context_key, *_ = key_error.args
```

Comment:
I had no idea `*` could be used on the left-hand side like this.

Reply:
😮

Reply:
Yeah. It's a python3-ism.